src/client/app/+global-settings/+general-settings/general-settings.component.html

```diff
@@ -0,0 +1,86 @@
+                        </ul>
+                        <div class="row" style="display:none;">
+                            <div class="col-xs-12 col-sm-5">
+                                <div class="form-group">
+                                    <input type="text" class="form-control" id="basicInput" placeholder="Industy">
+                                </div>
+                            </div>
+                            <div class="col-xs-12 col-sm-2">
+                                <button type="button" class="btn bg-orange">Add New</button>
+                            </div>
+                        </div>
+
+                        <div class="col-xs-12 col-md-8 col-lg-8" style="padding:0; float:none;margin: 0 auto;">
+                            <div class="box box-without-bottom-padding ">
+                                <div class="row">
+                                    <div class="col-xs-12 col-md-6 col-lg-6">
+
+                                        <label for="search" style="line-height: 47px; width:100%;">Search:
+                                            <input id="search" style="float:right;    width: 83%;" class="form-control" type="text" [(ngModel)]="search"/>    
+                                        </label>
+
+                                    </div>
+                                    <div class="col-xs-12 col-md-6 col-lg-6">
+
+                                        <label for="showNumberOfItems" style="line-height: 47px;">Show:
+                                            <select  id="showNumberOfItems" style="float:right; padding-top:10px;  width: 65%;" class="form-control" [(ngModel)]="showNumberOfItems" (ngModelChange)="changeRange($event)">
+                                                <option [value]=10>10</option>
+                                                <option [value]=15>15</option>
+                                                <option [value]=20>20</option>
+                                                <option [value]=25>25</option>
+                                                <option [value]=100>100</option>
+                                            </select>
+                                        </label>
+
+                                    </div>
+                                </div>
+                                <table class="table">
+                                   <thead>
+                                        <tr>
+                                            <th>SNO</th>
+                                            <th  *ngFor="let key of keys;let i=index" (click)="orderBy(key,i);">
+                                                <span style="display: inline-flex;">{{ key|uppercase}}
+                                                <span style="padding-left: 20%;float:right" *ngIf="resource.order[key]!='asc'&& resource.order[key]!='desc'&&sortedCols.indexOf(i)>-1;" class="glyphicon glyphicon-sort"></span>
```

Comment:
dont use any inline styles , if you tend to use something is required please get in touch with KARTHIK